### DISCUSSION TITLE
최대인원 설정하기

### DISCUSSION THREAD

**Discussion by hyeona-y** [naverz/zepeto-studio-kor#1143] [5. Q&A]
안녕하세요~
제페토 월드 세팅에서 최대인원을 16명으로 설정했는데 QR코드로 들어가서 확인할때는 8명으로 떠있습니다ㅠㅠ
서버에서는 최대수용인원이 16명으로 되어있는데 왜 숫자가 다르게 뜰까요?
밑에 관련 스샷 올립니다

![화면 캡처 2022-08-09 112624](https://user-images.githubusercontent.com/106052215/183550028-01f02326-1821-4e03-8702-1cdcf596bca0.png)
![Screenshot_20220809-112338_ZEPETO](https://user-images.githubusercontent.com/106052215/183550045-203a710b-bf09-4e39-8b98-ef7e3c927ad0.jpg)
![image (1)](https://user-images.githubusercontent.com/106052215/183550503-acdb168b-cc13-4f05-8bd6-f226984d2840.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 제보 감사합니다.
실제 접속 인원은 16명이 맞는데 UI 가 반영되지 않는 부분은 버그로 처리하겠습니다.
이슈 관리를 위해 https://github.com/naverz/zepeto-studio-kor/issues 이곳에 양식에 맞춰서 올려주시면 감사 드리겠습니다.
